event-streams/src/main/java/com/redhat/cloud/platform/playbook_dispatcher/RunEventTransform.java

```diff
@@ -143,6 +144,28 @@ Payload buildRunPayload(Struct input) {
         payload.setCreatedAt(input.getString("created_at"));
         payload.setUpdatedAt(input.getString("updated_at"));
 
+        if (input.get("name") != null) {
+            payload.setPlaybookName(input.getString("name"));
+        }
+        if (input.get("web_console_url") != null) {
+            payload.setPlaybookRunUrl(URI.create(input.getString("web_console_url")));
+        }
+
+        RecipientConfig recipientConfig;
+        try {
+            recipientConfig = objectMapper.readValue(input.getString("recipient_config"), RecipientConfig.class);
```

Comment:
There's not such field in the db. You'll need to create the recipient config object from reading the two plain top-level fields (sat_id and sat_org_id).